### ISSUE TITLE
[React native] null is not an object(evaluating 'this.room.myroomjid') 

### ISSUE THREAD

**Issue by vantuan88291** [jitsi/lib-jitsi-meet#2275]
## Steps to reproduce
---
When i join a room, this issue happened sometime if in a room have more than 2 people
# Environment details
---
v1506.0.0+3fc03265

**Comment by saghul** [comment]
Can you paste a trace back? Any chance you can test a more recent LJM version?

**Comment by AkshayaFrontM** [comment]
Any updates on this? I am facing the same issue. 

I can only get this from Stacktrace:

`node_modules/lib-jitsi-meet/dist/umd/lib-jitsi-meet.min.js:2
null is not an object (evaluating 'this.room.myroomjid')`

**Comment by saghul** [comment]
Are you using our RN SDK or using the library directly?

**Comment by Shalini0001** [comment]
I'm encountering the same error on both Android and iOS platforms:

Error: TypeError: null is not an object (evaluating 'this.room.myroomjid')

I'm using the following versions in my project:

"react-native": "0.72.9"
"lib-jitsi-meet": "https://github.com/jitsi/lib-jitsi-meet/releases/download/v1847.0.0+03eef6de/lib-jitsi-meet.tgz"

Can you please assist with resolving this issue? @saghul

**Comment by saghul** [comment]
Do you have a full backtrace?

**Comment by Shalini0001** [comment]
yes, I am using RN SDK,
node_modules/lib-jitsi-meet/dist/umd/lib-jitsi-meet.min.js:2 null is not an object (evaluating 'this.room.myroomjid')

**Comment by saghul** [comment]
What RN SDK version are you using?

**Comment by Shalini0001** [comment]
2.2.1

**Comment by Shalini0001** [comment]
Please reply @saghul

**Comment by saghul** [comment]
Please update to the latest version.

**Comment by Shalini0001** [comment]
If I updated with latest version then this error goes out ?

**Comment by saghul** [comment]
I believe it was fixed yes.

**Comment by Shalini0001** [comment]
Ok, I will try. Thankyou for your time.

**Comment by Shalini0001** [comment]
I used the latest version of jitsi-meet-sdk from https://github.com/jitsi/jitsi-meet.git, with Node.js version v20.17.0, along with "react": "18.2.0" and "react-native": "0.73.8". However, when I compile the code, the app installs on the device but displays an error on the screen.
![1725519548860](https://github.com/user-attachments/assets/9eb45670-ef39-4edc-b478-a4f322bbee0a)

**Comment by saghul** [comment]
Please try using the released package first.

**Comment by Shalini0001** [comment]
ok... But Ihave to used sdk

**Comment by saghul** [comment]
Use this package: https://www.npmjs.com/package/@jitsi/react-native-sdk

**Comment by Shalini0001** [comment]
I want to customised jitsi-meet-sdk for our requirements. I cannot used package.. Please help

**Comment by saghul** [comment]
First make sure things work with an unmodified version. Otherwise we don't know if you caused the problem or if it was there before.

**Comment by Shalini0001** [comment]
Yes, I try with unmodified version. but did not find what the issue it is 

![1725519548860](https://github.com/user-attachments/assets/943a46ed-e144-4697-9de6-5b2130d97258)

**Comment by saghul** [comment]
Have you tried the sample app?

**Comment by Shalini0001** [comment]
yes... Clone this repo https://github.com/jitsi/jitsi-meet.git and try to compile the project

**Comment by saghul** [comment]
Here is the sample app: https://github.com/jitsi/jitsi-meet-sdk-samples/tree/master/react-native

**Comment by Shalini0001** [comment]
Now I used the latest version of jitsi-meet-sdk from https://github.com/jitsi/jitsi-meet.git. but getting same error
![error](https://github.com/user-attachments/assets/b7cb2898-f4fb-4eda-beaa-954b7a03e866)
